client/src/components/pages/common/errors/Error-404/index.js

```diff
@@ -1,5 +1,11 @@
 import React from 'react';
+import Image from '../../../../common/Image/index';
+import ErrorImage from '../../../../../assets/images/error404.jpg';
 
-const Error = () => <h1>Error 404</h1>;
+const Error404 = () => (
+  <div>
+    <Image alt="Error 404" src={ErrorImage} style={{ marginTop: '3rem' }} />
```

Comment:
there is scroll in the page , and the image was large so try to make it smaller & without scroll, its better to create css file & fix these things using it.